src/tr/araznovel/src/eu/kanade/tachiyomi/extension/tr/araznovel/ArazNovel.kt

```diff
@@ -11,21 +11,12 @@ class ArazNovel : Madara("ArazNovel", "https://www.araznovel.com", "tr", SimpleD
 
     override fun chapterListParse(response: Response): List<SChapter> {
         val document = response.asJsoup()
-        val mangaId = document.select("div#manga-chapters-holder").attr("data-id")
+        val mangaUrl = document.location().removeSuffix("/")
 
-        val xhrRequest = oldXhrChaptersRequest(mangaId)
+        val xhrRequest = xhrChaptersRequest(mangaUrl)
         val xhrResponse = client.newCall(xhrRequest).execute()
 
-        return xhrResponse.asJsoup().let { xhrDocument ->
-            xhrDocument.select("li.parent").let { elements ->
-                if (!elements.isNullOrEmpty()) {
-                    elements.reversed()
-                        .map { volumeElement -> volumeElement.select(chapterListSelector()).map { chapterFromElement(it) } }
-                        .flatten()
-                } else {
-                    xhrDocument.select(chapterListSelector()).map { chapterFromElement(it) }
-                }
-            }
-        }
+        val chapterElements = xhrResponse.asJsoup().select("ul.version-chap li.wp-manga-chapter")
+        return chapterElements.map(::chapterFromElement)
     }
 }
```

Comment:
this is essentially the implementation of `chapterListParse` in the `Madara` class, so remove the override and check.

would also need to set `useNewChapterEndpoint` to true and adjust `chapterListSelector`

Reply:
turns out they use the old endpoint... and selector is the same....
![image](https://github.com/user-attachments/assets/61259bb3-2b3f-464e-a47c-996a6ea90338)